src/helpers/propOr.js

```diff
@@ -0,0 +1,25 @@
+/** @license ISC License (c) copyright 2017 original and current authors */
+/** @author Ian Hofmann-Hicks (evil) */
+
+const curry = require('../core/curry')
+const isNil= require('../core/isNil')
+const isInteger = require('../core/isInteger')
+const isString = require('../core/isString')
+
+const lift = (x, def) =>
+  isNil(x) ? def : x
+
+// propOr : a -> String | Number -> b -> c
```

Comment:
should be `// propOr : a -> String | Integer -> b -> c`

Reply:
Yep, I thought was the same. It needs to change on `prop` and `propPath` too, I take from there 😅